### ISSUE TITLE
Schematic

### ISSUE THREAD

**Issue by Speshman** [73Volvo/Speedypal#1]
Is there a schematic available please

**Comment by 73Volvo** [comment]
I have some gerber files, would that help? I'll try to upload them.

**Comment by Speshman** [comment]
Yep that would help. Do you have the schematic from kicad/eagle or whatever you designed in too?

**Comment by 73Volvo** [comment]
Sure. I used Easy EDA, and I just uploaded them.
